components/nexus-orient/src/main/java/org/sonatype/nexus/orient/entity/EntityAdapter.java

```diff
@@ -256,8 +255,7 @@ protected void setMetadata(final T entity, final ODocument doc) {
    * Get metadata from entity.
    */
   protected EntityMetadata getMetadata(final T entity) {
-    checkNotNull(entity != null);
-    assert entity != null;
+    checkNotNull(entity);
```

Comment:
ftr the assert here was probably to keep IDEA from complaining... I've yet to figure out how to make it learn what Preconditions do :-(